src/platform/user/profile/vap-svc/components/base/VAPServiceTransactionErrorBanner.jsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
+
+// eslint-disable-next-line deprecate/import
```

Comment:
I am temporarily disabling this `AlertBox` deprecation error because this is outside of the scope of the work here. We will be addressing replacing these with web components in the following issues:

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1917